src/applications/mhv-secure-messaging/tests/actions/draftDetails.unit.spec.jsx

```diff
@@ -107,6 +107,7 @@ describe('draftDetails actions', () => {
       .then(() => {
         expect(store.getActions()).to.deep.include({
           type: Actions.Thread.DRAFT_SAVE_STARTED,
+          payload: { messageId: undefined },
```

Comment:
looks like we should be expecting that payload to have a valid `messageId` from what I can tell looking in `actions/draftDetails.js:91`